packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -63,28 +63,34 @@ export interface DatepickerProps extends UIComponentProps, Partial<ICalendarStri
    */
   onDateChange?: ComponentEventHandler<DatepickerProps & { value: Date }>;
 
+  /**
+   * Called on change of the date.
+   *
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props and proposed value.
+   */
+  onDateEntryError?: ComponentEventHandler<DatepickerProps & { error: string }>;
```

Comment:
```suggestion
  onDateChangeError?: ComponentEventHandler<DatepickerProps & { error: string }>;
```

or

```suggestion
  onDateValidateError?: ComponentEventHandler<DatepickerProps & { error: string }>;
```
WDYT?

Reply:
Great proposals!
I think the first one is more explicit and in alignment with what we have for change event handler :)